pl_examples/domain_templates/generative_adversarial_net.py

```diff
@@ -127,12 +128,12 @@ def training_step(self, batch, batch_idx, optimizer_idx):
             # adversarial loss is binary cross-entropy
             g_loss = self.adversarial_loss(self.discriminator(self(z)), valid)
             tqdm_dict = {'g_loss': g_loss}
-            output = OrderedDict({
-                'loss': g_loss,
-                'progress_bar': tqdm_dict,
-                'log': tqdm_dict
-            })
-            return output
+            result = TrainResult(
```

Comment:
i would also add checkpoint_on in this case since there is no val loop

Reply:
I will add that.